app/src/main/java/org/openobservatory/ooniprobe/adapter/TestsListAdapter.java

```diff
@@ -42,58 +47,75 @@ public TestsListAdapter(MainActivity context, ArrayList<NetworkMeasurement> valu
     public TestsListAdapter.ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
         View v;
         switch (viewType) {
-            case 0:
-                v = LayoutInflater.from(parent.getContext()).inflate(R.layout.row_pending_test, parent, false);
-                break;
-            case 1:
-                v = LayoutInflater.from(parent.getContext()).inflate(R.layout.row_finished_test, parent, false);
-                break;
             default:
-                v = LayoutInflater.from(parent.getContext()).inflate(R.layout.row_pending_test, parent, false);
+                v = LayoutInflater.from(parent.getContext()).inflate(R.layout.row_finished_test, parent, false);
```

Comment:
Here I'd probably remove the switch (or perhaps you have in mind other changes that will require again the switch and in such case I'd add a comment to say that)